src/charts/sparkline.js

```diff
@@ -8,9 +8,9 @@ define(function(require){
     const d3Selection = require('d3-selection');
     const d3Transition = require('d3-transition');
 
-    const {exportChart} = require('./helpers/export');
+    const { exportChart } = require('./helpers/export');
     const colorHelper = require('./helpers/color');
-    const {line} = require('./helpers/load');
+    const { stackedArea } = require('./helpers/load');
```

Comment:
Seems a bit confusing to leave the name of the loader as `stackedArea`. I would recommend something like:
```javascript
const { stackedArea: stackedAreaLoader } = require('./helpers/load');
```
Just to make it clear that it's a loading state not the chart itself.

Reply:
I was super close to do that, you just gave me the final push!
Thanks!